packages/client/src/Dashboard.tsx

```diff
@@ -280,100 +281,114 @@ type Conversation = NonNullable<
-                    navigate(`/accounts/${accountId}/conversations/${id}`)
-                  }
-                  selected={isSelected}
-                >
-                  <ListItemAvatar>
-                    {isSelected ? (
-                      <MuiAvatar
-                        role="checkbox"
-                        aria-checked="true"
-                        aria-labelledby={rowId}
-                        className={classes.selectedAvatar}
-                        onClick={event => {
-                          event.stopPropagation()
-                          dispatch(Sel.unselect(id))
-                        }}
-                      >
-                        <CheckIcon />
-                      </MuiAvatar>
-                    ) : (
-                      <Avatar
-                        role="checkbox"
-                        aria-checked="false"
-                        aria-labelledby={rowId}
-                        onClick={event => {
-                          event.stopPropagation()
-                          dispatch(Sel.select(id))
-                        }}
-                        address={from}
-                      />
-                    )}
-                  </ListItemAvatar>
-                  <ListItemText
-                    id={rowId}
-                    primary={subject || "[no subject]"}
-                    secondary={
-                      <>
-                        <Typography
-                          component="span"
-                          variant="body2"
-                          className={classes.inline}
-                          color="textPrimary"
-                        >
-                          {moment(date).calendar()}
-                          {"\u00A0"}
-                          from{" "}
-                          {from
-                            ? from.name || `${from.mailbox}@${from.host}`
-                            : "unknown sender"}
-                        </Typography>
-                        {` — ${snippet}`}
-                      </>
-                    }
-                    classes={{
-                      secondary: classes.snippetText
-                    }}
-                  />
-                </ListItem>
-                {index === conversations.length - 1 ? (
-                  ""
-                ) : (
-                  <Divider variant="inset" component="li" />
-                )}
-              </React.Fragment>
-            )
-          }
-        )}
+        {conversations.map((conversation, index) => {
+          return (
+            <React.Fragment key={conversation.id}>
+              <ConversationRow conversation={conversation} {...rest} />
+              {index === conversations.length - 1 ? (
+                ""
+              ) : (
+                <Divider variant="inset" component="li" />
+              )}
+            </React.Fragment>
+          )
+        })}
       </List>
     </Paper>
   )
 }
 
+function ConversationRow({
```

Comment:
I split out a new component to display each row in the conversation list view. This way I can easily locate conversation rows in tests with `app.find("ConversationRow")`.